recipes/image_classification/classification.py

```diff
@@ -113,7 +113,7 @@
 # Dataset parameters
 group = configlib.add_parser("Dataset parameters")
 # Keep this argument outside of the dataset group because it is positional.
-group.add_argument("data_dir", metavar="DIR", help="path to dataset")
+group.add_argument("--data_dir", metavar="DIR", help="path to dataset")
```

Comment:
this might break something; let's check if we need to make some extra changes (e.g. to the classification README.md)